tests/system_test.py

```diff
@@ -926,15 +926,15 @@ def get_queue_stats(self):
         return self.queue_stats % (self.num_queue_puts, self.num_queue_gets)
 
     def _main(self):
-        self._container.timeout = 0.5
+        self._container.timeout = 3.0
```

Comment:
Maybe use a global constant for the three seconds? So that the timeouts can be unified across tests, based on what they "mean". Something like `DEFAULT_TEST_CLIENT_CONTAINER_TIMEOUT = 3`?

Reply:
In addition to system_tests.TIMEOUT which is there already, https://github.com/apache/qpid-dispatch/blob/138d084d7a0d9dbeed2991affffc8f669563be68/tests/system_test.py#L124-L126